packages/website/pages/[...slug].tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import ErrorPage from 'next/error';
+import rehypeAutolinkHeadings from 'rehype-autolink-headings';
+import rehypeSlug from 'rehype-slug';
+import mdxPrism from 'mdx-prism';
+import { css } from 'emotion';
+import { serialize } from 'next-mdx-remote/serialize';
+import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote';
+import Link from 'next/link';
+
+import { getMdxPaths, getMdxSourceBySlug } from '../utils/content';
+
+const styles = {
+  root: css({
+    minHeight: '100vh',
+    padding: '0 0.5rem',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  }),
+};
+
+type ComponentPageProps = {
+  source: MDXRemoteSerializeResult;
+  frontMatter: {
+    title: string;
+  };
+};
+
+export default function ComponentPage(props: ComponentPageProps) {
```

Comment:
because slug can have more than one segment: `/components/skeleton/skeleton-image/`

`[...slug]` means that slug is an array. in this case `["components", "skeleton", "skeleton-image"]`

Reply:
ohhhhhhhhh I totally missed that! Thx